libfwupd/fwupd-enums.h

```diff
@@ -64,6 +64,24 @@ typedef enum {
 	FWUPD_TRUST_FLAG_LAST
 } FwupdTrustFlags;
 
+/**
+ * FwupdFeatureFlags:
+ * @FWUPD_FEATURE_FLAG_NONE:			No trust
+ * @FWUPD_FEATURE_FLAG_CAN_REPORT:		Can report successful update
```

Comment:
presumably not just successful updates?

Reply:
`Can upload a report of the update back to the server`?

Reply:
ya